sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_file_system_client_async.py

```diff
@@ -385,7 +384,7 @@ def get_paths(self, path=None,  # type: Optional[str]
                         recursive=True,  # type: Optional[bool]
                         max_results=None,  # type: Optional[int]
                         **kwargs):
-        # type: (...) -> ItemPaged[PathProperties]
+        # type: (...) -> AsyncItemPaged[PathProperties]
```

Comment:
isn't this a break? what if somebody checks type? should we alias?

Reply:
We have previously fixed incorrect type annotations. The return object in this method is `AsyncItemPaged` so this was a mistake that needed to be fixed.